CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 ----------
 
+[3.2.0] - 2022-11-30
+--------------------
+Changed
+~~~~~~~
+* Add event_metadata parameter to EventBusProducer send API.  **Note:** Calling this method with the new argument will throw an exception unless the EventBusProducer implementation used also upgraded to support the argument.
```

Comment:
Nit: I think you are saying the following? If so, maybe this comment is more clear? You be the judge.
```suggestion
* Add event_metadata parameter to EventBusProducer send API.  **Note:** Calling this method with the new argument will throw an exception with older versions of this library that don't yet support the parameter.
```

Reply:
I reviewed the wrong PR. :)